src/applications/claims-status/actions/index.jsx

```diff
@@ -193,7 +200,8 @@ export function getSyncStatus(claimsAsyncResponse) {
   return get('meta.syncStatus', claimsAsyncResponse, null);
 }
 
-export function getClaimsV2(poll = pollRequest) {
+export function getClaimsV2(options = {}) {
+  const { poll = pollRequest, pollingExpiration } = options;
```

Comment:
I wanted to make it possible to pass in the new optional `pollingExpiration` _without_ explicitly passing in the `pollRequest`. So I switched to using a single options object. Now you can do `getClaimsV2({pollingExpiration: 1000})` and this function will still use `pollRequest` as the default value when we don't explicitly pass in a `poll` option.

Reply:
example error output:
<img width="725" alt="Screen Shot 2021-04-13 at 3 37 28 PM" src="https://user-images.githubusercontent.com/20728956/114630958-8a919900-9c70-11eb-894c-94ead72f314e.png">